processors/data_contributions/extractors.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+
+def extract_source():
+    return {
+        'id': 'contrib',
```

Comment:
No need to abbreviate here. Use `data_contribution` or simply `contribution` to improve readability.